package.json

```diff
@@ -12,8 +12,8 @@
     "preview": "vite preview",
     "docs": "jsdoc --readme ./docs/README.md -c jsdoc.config.json",
     "docs:clear": "node clearDocs.js",
-    "lint": "eslint 'src/**/*.js' 'src/**/*.jsx' 'test/**/*.js' 'test/**/*.jsx'",
-    "lint:fix": "eslint --fix 'src/**/*.js' 'src/**/*.jsx' 'test/**/*.js' 'test/**/*.jsx'",
+    "lint": "eslint 'src/**/*.js' 'src/**/*.jsx' 'test/**/*.js' 'test/**/*.jsx' '__mocks__/**/*.js'",
```

Comment:
A merge issue caused these mocks to start erroring. We can have eslint alert us to the issue ahead of time, and avoid some headache.